src/main/java/io/debezium/connector/db2/Db2ChangeTable.java

```diff
@@ -30,16 +30,26 @@
      */
     private Lsn stopLsn;
 
+    /**
+     * The table in the CDC schema that captures changes, suitably quoted for Db2
+     */
+    private final String db2CaptureInstance;
```

Comment:
@lga-zurich, made this one final; other than that, it's exactly like your original PR.

Reply:
Sounds good to me! Cheers @gunnarmorling